<ui version="4.0" >
 <class>qt4genpngwnd</class>
 <widget class="QWidget" name="qt4genpngwnd" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Gen PNG Options</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QPushButton" name="pbNormalFont" >
     <property name="minimumSize" >
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <family>Arial</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>&amp;Normal font</string>
     </property>
     <property name="shortcut" >
      <string>Alt+N</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbItalicFont" >
     <property name="minimumSize" >
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <family>Arial</family>
       <pointsize>28</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text" >
      <string>&amp;Italic font</string>
     </property>
     <property name="shortcut" >
      <string>Alt+I</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbBoldFont" >
     <property name="minimumSize" >
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <family>Arial</family>
       <pointsize>28</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>&amp;Bold font</string>
     </property>
     <property name="shortcut" >
      <string>Alt+B</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Norm</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbNTSC" >
        <property name="text" >
         <string>N&amp;TSC (720x480 )</string>
        </property>
        <property name="shortcut" >
         <string>Alt+T</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbPAL" >
        <property name="text" >
         <string>&amp;PAL ( 720x576 )</string>
        </property>
        <property name="shortcut" >
         <string>Alt+P</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1_2" >
       <property name="minimumSize" >
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Horizontal alignment</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>leBaseName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbHoriz" >
       <item>
        <property name="text" >
         <string>Center</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Left</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Right</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1_2_2" >
       <property name="minimumSize" >
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Vertical alignment</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>leBaseName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbVert" >
       <item>
        <property name="text" >
         <string>Bottom</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Top</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1_2_2_2" >
       <property name="minimumSize" >
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Left margin</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>leBaseName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbLeftMargin" >
       <property name="value" >
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1_2_2_2_2" >
       <property name="minimumSize" >
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Right margin</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>leBaseName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbRightMargin" >
       <property name="value" >
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1_2_2_2_3" >
       <property name="minimumSize" >
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Top mar&amp;gin</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>leBaseName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbTopMargin" >
       <property name="value" >
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1_2_2_2_4" >
       <property name="minimumSize" >
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Botto&amp;m margin</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>leBaseName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbBottomMargin" >
       <property name="value" >
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="pbSubColor" >
       <property name="text" >
        <string>Subtitle color</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbForced" >
     <property name="text" >
      <string>Forced subtitles</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="minimumSize" >
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Ba&amp;se name of png</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>leBaseName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leBaseName" >
       <property name="text" >
        <string>sub</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>91</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk" >
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Ok</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel" >
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
